daemon/algod/api/server/v2/utils.go

```diff
@@ -67,6 +70,29 @@ func notImplemented(ctx echo.Context, internal error, external string, log loggi
 	return returnError(ctx, http.StatusNotImplemented, internal, external, log)
 }
 
+func convertSlice[X any, Y any](input []X, fn func(X) Y) []Y {
+	output := make([]Y, len(input))
+	for i := range input {
+		output[i] = fn(input[i])
+	}
+	return output
+}
+
+func uint64Slice[T ~uint64](s []T) []uint64 {
+	return convertSlice(s, func(t T) uint64 { return uint64(t) })
+}
+
+func stringSlice[T interface{ String() string }](s []T) []string {
```

Comment:
Up to you but, fmt.Stringer exists if you want to use it:
```suggestion
func stringSlice[T fmt.Stringer](s []T) []string {
```

Reply:
Thanks for pointing this out. Done in e423c7d2f